### ISSUE TITLE
First Entry in Endpoints not showing

### ISSUE THREAD

**Issue by VidaRD1234** [CiscoDevNet/iPSK-Manager#35]
**Describe the bug**
After Adding an Endpoint the GUI is not showing the entry.
![grafik](https://github.com/CiscoDevNet/iPSK-Manager/assets/144127014/2f47260f-9c1d-4df4-9869-9502701026cf)

The entry can be seen in the database
![grafik](https://github.com/CiscoDevNet/iPSK-Manager/assets/144127014/6e82edff-e2bd-4625-b004-38f44cb3e3e0)

After adding a second entry the one before it will show up.
So basically I am always missing the newest entry.

**Server:**
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04]

**Browsers:**
 - Firefox Version 115.7.0esr (32-Bit)
 - Edge Version 121.0.2277.128 (64-Bit)

Any help would be appreciated

**Comment by VidaRD1234** [comment]
Just noticed this was fixed. Thx in advance
